packages/accounts/src/utils/balance-id.ts

```diff
@@ -83,6 +84,10 @@ export function toSettlementId({
 //   )
 // }
 
+export function bigIntToUuid(id: bigint): Pojo {
+  return raw('?::uuid', [id.toString(16).padStart(32, '0')])
+}
+
```

Comment:
Ideally we want to get this not having to call postgres to do the conversion

Reply:
https://github.com/coilhq/rafiki/pull/51/commits/7a2934ad7c09842ebe811ff6e8edf050049b110d limits this to `$formatDatabaseJson` where the typecast is added to the postgres query
https://vincit.github.io/objection.js/api/model/overview.html#model-data-lifecycle

Reply:
Ah interesting, thats pretty neat!